config/versions/versions.go

```diff
@@ -27,16 +28,19 @@ import (
 )
 
 // LatestVersion used as version param to Deploy to automatically use the latest version
-const LatestVersion = -1
+const LatestVersion = math.MaxUint16
```

Comment:
Given is not written to with the latest version, is not used to store the latest version, is a const, I think it would be more appropriately named as `MaxVersion`

Reply:
`MaxVersion` for me would be the highest possible version.
Which is `math.MaxUint16 -1`
Since `LatestVersion` is used to say "automatically use the latest version", `UseLatestVersion` might be the right compromise.

Fixed https://github.com/gbjk/gocryptotrader/commit/c4d846e059